packages/rest/src/http/decorators.ts

```diff
@@ -594,7 +601,7 @@ export function getRouteOptionsForNamespace(
 }
 
 const routesKey = createStateSymbol("routes");
-function setRoute(context: DecoratorContext, entity: Type, details: RoutePath) {
+function setRoute(context: DecoratorContext, entity: Type, details: RoutePath, parameters?: Model) {
```

Comment:
`parameters` is not used here

Reply:
Yep, it's part of RoutePath. Will remove.